src/Sound/SoundPack.cpp

```diff
@@ -74,7 +92,7 @@ void SoundPack::LoadFromFile(const std::string& filePath)
 				sound.volume = volume;
 				sound.pitch = pitch;
 				sound.pitchDeviation = sample.pitchDeviation;
-				sound.bytes = soundData;
+				sound.bytes = std::vector<uint8_t>(soundData.begin(), soundData.end());
```

Comment:
Just so you know, this is a copy of the span into separate vectors. If you want to take advantage of spans you'll have to store the larger buffer as a copy and copy the spans

Reply:
I didn't do this here because it would be out of scope